airflow/utils/helpers.py

```diff
@@ -294,6 +294,21 @@ def render_template_as_native(template: jinja2.Template, context: Context) -> An
     return render_template(template, cast(MutableMapping[str, Any], context), native=True)
 
 
+def at_least_one(*args) -> bool:
```

Comment:
Is it really necessary? I would use `any` directly. This function does not add any value from `any`. It just checks the values of the parameters. But I guess `any` throws also an exception in such case.

Reply:
This function makes the test fail because this code is part of core Airflow but is used in the Amazon provider package. Core Airflow and Amazon provider package are releases separately, therefore you add a dependency between the two packages. There is a solution for that, but as mentioned in my previous comment, this function is useless to me.

Reply:
See my convo above, it was added because of my comment.